tape/src/main/java/com/squareup/tape2/QueueFile.java

```diff
@@ -638,4 +627,40 @@ public void clear() throws IOException {
           + "]";
     }
   }
+
+  /** Fluent API for creating {@link QueueFile} instances. */
+  public static class Builder {
+    File file;
+    Boolean zero;
+
+    /** The underlying {@link File} backing this queue. */
+    public Builder file(File file) {
```

Comment:
forbid null. also put this in the constructor of the builder? it's a queueFILE, after all.